docs/man.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+has_children: true
+nav_order: 4
+has_toc: false
+---
+
+# Command line reference
```

Comment:
I think it would be better to have only the support commands in this user docpage. The others are not really aimed at users, and may contribute in confusing them.

If we want to also cover them for our reference, we can add a few minor pages like `osmet.md` and `rdcore.md` under the "development" section.